src/core/Button/Button.js

```diff
@@ -3,27 +3,17 @@ import PropTypes from 'prop-types'
 import Icon from '../Icon'
 import s from './styles'
 
-function Button({
-    role,
-    type,
-    size,
-    kind,
-    icon,
-    label,
-    active,
-    disabled,
-    onClick,
-}) {
+function Button({ role, type, kind, icon, label, active, disabled, onClick }) {
```

Comment:
I don't remember the use-case for *active*. I just copied your earlier code here

Reply:
I don't see a use case for it in Joe's designs. Second to remove _active_.